src/span.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+import * as Constants from './constants';
```

Comment:
This is [ES6 syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import). 

The JS is being compiled with [Babel](https://babeljs.io/) so there's less concern about what particular browser or Node version is being used by the consumer of the code.